google-cloud-bigtable/scripts/regenerate.sh

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env bash
+
+# This script is used to regenerate protos & GAPIC stubs.
+# It stages all of the generated code in deps/ and then installs it into the local maven repo.
+# It is meant to be used to test changes to the gapic config before publishing
+# To pull in upstream changes, please use update-deps.sh
+
+set -eE # same as: `set -o errexit -o errtrace`
```

Comment:
If you're going to spill the ink to say what the command is, why not just use the extended command? :-P